pandas/tests/series/methods/test_append.py

```diff
@@ -61,6 +61,16 @@ def test_append_tuples(self):
 
         tm.assert_series_equal(expected, result)
 
+    def test_append_dataframe_raises(self):
```

Comment:
this test should now be a regression test that shows that the bahviour is unchanged from 0.25.3